carrierwave_backgrounder.gemspec

```diff
@@ -3,12 +3,12 @@ $:.push File.expand_path("../lib", __FILE__)
 require "backgrounder/version"
```

Comment:
Rename the gemspec file, and add a file `lib/carrierwave/activejob.rb` which just does `require "carrierwave_backgrounder"`. I think that's necessary that the gem can be required without manually specifying `require` in the Gemfile.

Reply:
You can test if this actually works by pointing your Gemfile to this branch.

Reply:
I mounted my local branch and uploaded a showreel video, completed without error message. But when I enter `Carrierwave::Backgrounder` in the console, it says "uninitialized constant".

Reply:
When I remove the require statement in the gem, I get an error in showreel manager (undefined method `processed?`), so I think it works.

Reply:
Try `CarrierWave::Backgrounder`? But if there's no error when uploading videos it has to work.